pkgs/data/fonts/alkalami/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, fetchzip }:
+
+let version = "2.000"; in
+
+fetchzip rec {
```

Comment:
While this clearly works, not sure if not using `stdenv.mkDerivation` or `stdenvNoCC.mkDerivation` instead here is a good idea.

Specially because since this lacks the `pname` and `version` attributes, this will probably break some of the tooling we have.

Reply:
This appears to be common practice in the `data/fonts/` directory.

Reply:
> Specially because since this lacks the `pname` and `version` attributes, this will probably break some of the tooling we have.

Supersandro fixed it already. Nowadays fetchzip supports pname & version.

Reply:
Yeah, I saw this later that fonts generally do this.

Would still be nice to use `pname` and `version` attributes though.